index.html

```diff
@@ -193,6 +193,59 @@ <h3>Skills</h3>
           <img src="imgs/tech-img-3.png" alt="" />
         </div>
       </section>
+
+      <footer class="contact">
+        <img src="imgs/footer-img.png" alt="" />
+        <div>
+          <h2>Get started</h2>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna.
+          </p>
+        </div>
+
+        <form action="https://formspree.io/f/xnqrodae" method="post">
+          <ul>
+            <li>
+              <label for="">Your name</label>
+              <input
+                type="text"
+                id="name"
+                name="username"
+                maxlength="30"
+                placeholder="Full name"
+                required
+              />
+            </li>
+            <li>
+              <label for="">Your email</label>
+              <input type="email" placeholder="Your email" />
+            </li>
+            <li>
+              <label for=""></label>
+              <textarea name="message" maxlength="500" cols="30" rows="10">
```

Comment:
- [ ] As required, please kindly ensure that all input fields have the `required` attribute to activate the validation on submit.

Reply:
Fixed.